### ISSUE TITLE
cannot assign requested address for ** port 53

### ISSUE THREAD

**Issue by balajeek** [MatthewVance/unbound-docker#73]
i just spun up ubound in anticipation of creating my own lan dns records.
i have pi-hole setup and using it for a while now.

I get error (repeat 3 times) nothing else on the log.  (first i got port 53 was used by pihole, so i switched to 5335)
```
[1628875978] libunbound[20:0] error: udp connect failed: Cannot assign requested address for 2001:500:200::b port 53
```

here is my docker-compose:
```
  unbound:
    image: mvance/unbound:latest
    container_name: unbound
    restart: unless-stopped
    volumes:  
      - home/bk/unbound/:/opt/unbound/etc/unbound/
    ports:
      - 5335:53/tcp
      - 5335:53/udp

```

Tried googling for last two hours, found similar reports but they say its been fixed long ago.
Any help is appretiated.  thanks.

**Comment by MatthewVance** [comment]
Please see https://github.com/MatthewVance/unbound-docker/issues/55.

**Comment by balajeek** [comment]
thanks for the quick response.

Isn't when i say :latest in the docker-compose it should've got the 1.13.1 which is latest!  I checked the log anyway and it says 1.13.1 is what i have, but after googling i found a post where one says to add "interface: ::0".  After adding It seem unbound is running atleast and the log file shows the following:

```
[1628885192] unbound[1:0] notice: init module 0: validator
[1628885192] unbound[1:0] notice: init module 1: iterator
[1628885192] unbound[1:0] info: start of service (unbound 1.13.1).
[1628886364] unbound[1:0] info: service stopped (unbound 1.13.1).
```

docker container still seem to show the "cannot assign requested address for ..." error.
Does this mean even with the error the unbound is running? how can i test it?

Also the primary goal for me to install unbound is to have local lan dns, so i could say "mycomputer.home" on the browser should take me to local ip address.  So i googled more and included "a-records.conf" which contains:

```
local-data: "mycomputer.home A 192.168.99.11"
```

this doesn't seem to work, 
At this point i am not sure if I be troubleshooting this or make sure my unbound first works.

thanks for your support.

**Comment by MatthewVance** [comment]
Hi @balajeek. Thanks for your patience. 

I think a couple of issues are going on here. 

For your local DNS records issue is likely due to a formatting issue. You're missing a period: `local-data: "mycomputer.home. A 192.168.99.11"`

The udp connect failed error is unrelated the issue with not being able to create your own LAN DNS records. For background, the following error has been an ongoing issue for me. I've seen reports of it being fixed upstream earlier this year, but that never resolved it for me in Docker. Other than those messages occurring in the logs, Unbound in Docker has continued to work without issue. I'm hopeful [bug fix #422](https://github.com/NLnetLabs/unbound/issues/422) noted in the recently released [1.13.2 change log](https://nlnetlabs.nl/projects/unbound/download/#unbound-1-13-2) fully resolves this.

**Comment by MatthewVance** [comment]
@balajeek I'm going to go ahead and close this one because I haven't heard back in a while. If the above didn't solve it for you, feel free to reopen this.
